### PR TITLE
docs: fix inconsistent platform names

### PR DESCRIPTION
**Pull request by tong**
#### Description of Change
Fix inconsistent platform names.

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines]

#### Release Notes
Notes: no-notes

### CODE REVIEW

File: docs/api/app.md [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ details. Disabled by default.
 Set the about panel options. This will override the values defined in the app's
 `.plist` file. See the [Apple docs][about-panel-options] for more details.
 
-### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (mas)_
+### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS_
```
Comment (sindresorhus) [inline docs/api/app.md]: `(mas)` has a special meaning, as it indicates that this is only for Mac App Store (mas) apps. I do think that could be clearer though. I would just spell it out fully instead:

> _macOS_ (Mac App Store)

Comment (tong) [reply docs/api/app.md]: I see.
Thought it's a typo since it's the only occurrence.

Personally i would prefer a more programmatic naming like `_macOS-appstore_`.

For the haxe type definitions i am annotating fields with these platform flags to have compile time checks for supported api calls (like: https://github.com/tong/hxelectron/blob/75fc9cbc8a69b578f083780f181cd210ad97dbe4/src/electron/main/App.hx#L277).

A proper naming scheme would make it easier to pass such flags to the compiler as they could be the same as defined in the docs.

Strings like `_platform (App Store)_` are not very handy for this.
…  also for possible future flags (for example: `windows-appstore` instead of `windows (Windows App Store)`).

Guess other platforms would profit from this too.

File: docs/api/app.md [hunk 2]
```diff
@@ -1076,7 +1076,7 @@ details. Disabled by default.
 Set the about panel options. This will override the values defined in the app's
 `.plist` file. See the [Apple docs][about-panel-options] for more details.
 
-### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (mas)_
+### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (Mac App Store)_
```
Comment (MarshallOfSound) [inline docs/api/app.md]: This is gonna sound silly but this is a breaking change for our docs linter / parser / typescript generator.  I don't see how this is an "inconsistent" platform name when it is the **only** MAS only API.  Can we revert this change please?

File: docs/api/app.md (hunk 1 above)
Comment (MarshallOfSound) [reply docs/api/app.md]: > Personally i would prefer a more programmatic naming like _macOS-appstore_.

This aren't docs for programs, these are docs for humans 😄 Everything needs to be human readable without pre-processing.

The syntax 

```
_platform (restriction)_
```

Can we inferred and strictly qualified, in fact putting `mas` there makes it easier to enforce runtime / compiletime restrictions that you check `process[restriction]` (`process.mas`) before using the API if that's what you want to do.
